docs/in_production.rst

```diff
@@ -64,4 +67,32 @@ Configuring realtime data ingestion
 Monitoring Pinot
 ~~~~~~~~~~~~~~~~
 
+In order for Pinot to provide effective service there is a core set of metrics which should be monitored to ensure service stability, fault tolerance and acceptable response times. In the section following, there are service level metrics which are recommended to be monitored.
+
+More info on metrics collection and viewing a complete set of available metric is available in the `Metrics <customizations.html#metrics>`_ section.
+
+Pinot Server
+
+* Missing Segments - Number of missing segments - `NUM_MISSING_SEGMENTS <https://github.com/apache/incubator-pinot/blob/master/pinot-common/src/main/java/org/apache/pinot/common/metrics/ServerMeter.java>`_
```

Comment:
This explanation is confusing.

Metric is number of missing segments that the broker queried for (expected to be on the server) but the server didn't have. This can be due to retention or stale routing table.